app/src/main/java/org/thunderdog/challegram/ui/ProfileController.java

```diff
@@ -3835,6 +3835,24 @@ private void buildEditCells () {
       items.add(new ListItem(ListItem.TYPE_DESCRIPTION, 0, 0, R.string.AggressiveAntiSpamDesc));
     }
 
+    /*TdApi.SuggestedAction[] actions = tdlib.getSuggestedActions();
+    for (TdApi.SuggestedAction action : actions) {
+      UI.showToast(action.toString(), Toast.LENGTH_SHORT);
+      if (!tdlib.isBroadcastConvertSuggestion(action)) {
+        continue;
+      }
+      if (action.getConstructor() ==  TdApi.SuggestedActionConvertToBroadcastGroup.CONSTRUCTOR) {*/
+        boolean alwaysVisible = BuildConfig.DEBUG || BuildConfig.EXPERIMENTAL;
+        if (supergroup != null && supergroup.memberCount >= tdlib.supergroupMaxSize() - 50 /*little margin for leaving/joining members*/ || alwaysVisible && tdlib.canConvertToBroadcast(chat.id)) {
```

Comment:
This still ignores server suggestions completely. I've added new method in bd3596c, please use it
```suggestion
        boolean alwaysVisible = BuildConfig.DEBUG || BuildConfig.EXPERIMENTAL;
        if (alwaysVisible || tdlib.suggestConvertToBroadcastGroup(chat.id)) {
```